signac/contrib/schema.py

```diff
@@ -332,6 +338,11 @@ def difference(self, other, ignore_values=False):
             )
         return ret
 
+    @deprecated(
+        deprecated_in="1.8",
+        removed_in="2.0",
```

Comment:
Are we using `x.y` vs. `x.y.z` version numbers consistently in our deprecation notices?

Reply:
With the exception of JobsCursor.next being deprecated in 0.9.6 I think so.